plugins/flatpak/gs-flatpak.c

```diff
@@ -3984,6 +4062,12 @@ gs_flatpak_update_app (GsFlatpak *self,
 		}
 	}
 
+	if (!gs_flatpak_has_space_to_update (self, app, list)) {
+		g_debug ("Skipping automatic update for %s: not enough space on disk",
```

Comment:
The function is also used for manual updates (it will check the minimum space only). So this message is misleading. Maybe you should pass the `is_auto_update` as a parameter to the function (making it more clear), and use it here to filter two appropriate messages.